block.py

```diff
@@ -4,100 +4,103 @@
 
 from web3 import Web3
 
-from schemas import Block
+from schemas import Block, BlockCall, BlockCallType
 
 
 cache_directory = './cache'
 
 
-def get_transaction_hashes(calls: List[dict]) -> List[str]:
-    result = []
-
-    for call in calls:
-        if call['type'] != 'reward':
-            if call['transactionHash'] in result:
-                continue
-            else:
-                result.append(call['transactionHash'])
-    
-    return result
-
-
-def write_json(block: Block):
-    cache_path = _get_cache_path(block.block_number)
-    write_mode = "w" if cache_path.is_file() else "x"
-
-    with open(cache_path, mode=write_mode) as cache_file:
-        cache_file.write(block.json(sort_keys=True, indent=4))
-
-
 ## Creates a block object, either from the cache or from the chain itself
 ## Note that you need to pass in the provider, not the web3 wrapped provider object!
 ## This is because only the provider allows you to make json rpc requests
 def createFromBlockNumber(block_number: int, base_provider) -> Block:
```

Comment:
Oops, my bad for calling this `createFromBlockNumber` and not `create_from_block_number`

Reply:
not a show stopper

Reply:
I can update in a small PR